crates/node/src/chain_spec.rs

```diff
@@ -89,6 +89,17 @@ pub fn development_config(enable_manual_seal: Option<bool>) -> Result<DevChainSp
     ))
 }
 
+use sp_core::hexdisplay::HexDisplay;
+pub fn print_development_account() {
+    let argent_account_address = Felt252Wrapper::from_hex_be(ARGENT_ACCOUNT_ADDRESS).unwrap();
```

Comment:
we have a couple of accounts inside `crates/node/src/constants.rs`, 0x1 is the no validate account, 0x2 is the argent account, 0x3 is a OZ account, 0x4 is a no validate cairo 1 account, let's display all of them with these details